IdentityCore/src/requests/sdk/MSIDBrokerResponseHandler.m

```diff
@@ -166,9 +172,27 @@ - (MSIDTokenResult *)handleBrokerResponseWithURL:(NSURL *)response sourceApplica
                                                  accountMetadataCache:self.accountMetadataCacheAccessor
                                                         correlationID:correlationId
                                                      saveSSOStateOnly:brokerResponse.ignoreAccessTokenCache
+                                                           authScheme:authScheme
                                                                 error:error];
 }
 
+- (id<MSIDAuthenticationSchemeProtocol>)authSchemeFromResumeState:(NSDictionary *)resumeState
+{
+    NSMutableDictionary *schemeParams = [NSMutableDictionary new];
+    NSString *tokenType = resumeState[MSID_OAUTH2_TOKEN_TYPE];
+    NSString *requestConf = resumeState[MSID_OAUTH2_REQUEST_CONFIRMATION];
+    [schemeParams msidSetNonEmptyString:tokenType forKey:MSID_OAUTH2_TOKEN_TYPE];
+    [schemeParams msidSetNonEmptyString:requestConf forKey:MSID_OAUTH2_REQUEST_CONFIRMATION];
+    if ([schemeParams count] == 2)
```

Comment:
should we instead check for specific token type? In the future, we might have other token types that have 2 parameters.